db/migrate/20201215212934_add_mqtt_client_uuid_to_source.rb

```diff
@@ -0,0 +1,5 @@
+class AddMqttClientUuidToSource < ActiveRecord::Migration[5.2]
+  def change
+    add_column :sources, :mqtt_client_id, :uuid
```

Comment:
@hsong-rh I think this should just be a string since we wont know the format, looks like a guid but it is something external to us.

Reply:
Changed.